src/main/java/com/google/sps/data/PlaceReviews.java

```diff
@@ -73,9 +73,9 @@ private void replaceReview(Comment review) {
    * Assumes that there was a previous post by author
    * @return Comment the previous comment
    *                  TODO: Implement with Binary Search
+   *                        Collections.sort(this.reviews, Comment.ORDER_BY_AUTHOR);
```

Comment:
TODO has a lot of indent, remove unneccesary whitespace. Also, given time constraints, and low number of reviews, might not be worth doing binary search.